src/i18n/sv.json

```diff
@@ -176,7 +176,11 @@
     "measurementDisclaimerCollar": "Med undantag av kragmåttet, som är uppmätt på kragens insida.",
     "measurementDisclaimerInside": "Dessa mått är uppmätta <u>på insidan av artikeln</u>.",
     "tableTitle": "Aktuella artikel mått",
-    "windowTitle": "Storleks Guide för"
+    "windowTitle": "Storleks Guide för",
+    "tooltipBullet1": "!!Save your measurements so we will remember you next time!!!",
```

Comment:
@nomasi and here

Reply:
The Swedish translation isn't used, so let's fix this when we have a use-case.